src/python/grapl_e2e_tests/tests.py

```diff
@@ -222,4 +231,24 @@ def delete_model_plugin(
     assert deleted
 
 
-# ---------------------------- end model plugin helpers ------------------------------------
+# ---------------------------- AUTH HELPERS v ------------------------------------
+
+
+def get_notebook_url() -> None:
+    jwt = EngagementEdgeClient().get_jwt()
+    notebook_url = EngagementEdgeClient().get_notebook(jwt)
+
+    if (
+        "localhost:8888" in notebook_url
+    ):  # TODO: Need to conditionally change for AWS Deployments
+        assert f"Found notebook url on {notebook_url}"
+    else:
+        assert f"Unable to retrieve notebook url or notebook url is invalid: {notebook_url}"
```

Comment:
According to: https://www.8bitavenue.com/difference-between-assertion-and-exception-in-python/

"An assertion is a (debugging) boolean expression to self check an assumption about the internal state of a function or object. If true, nothing happens otherwise, execution halts and an error is thrown"

" An exception is an unexpected execution event that disrupts the normal flow of the program. Exceptions are used to signal an error condition where a corrective action can be taken (ex. file not found). Exceptions can be caught and recovered from as opposed to assertions."

Therefore, asserting valid test case and changed else: to exceptions